extensions/amp-analytics/0.1/visibility-model.js

```diff
@@ -298,7 +309,7 @@ export class VisibilityModel {
       return;
     }
     const conditionsMet = this.updateCounters_(visibility);
-    if (conditionsMet) {
+    if (conditionsMet || this.forceReport_) {
```

Comment:
nit: could we have `conditionsMet = this.updateCounters_(visibility) || this.forceReport_;` instead.  And comment please

Reply:
Done